### PR TITLE
Feature/optionality removal

### LINKED ISSUE #9: Remove optionality from TabDataSource's API
https://github.com/willowtreeapps/WillowTreeScrollingTabController/blob/master/Sources/ScrollingTabController.swift#L49

Feels awkward to be returning an optional here (and other similar methods). I don't see a valid case for this API choice.

### PR DESCRIPTION
**Pull request by eriklamanna**
Removed data source that's no longer needed.

Also fixes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ In addition, even more customization of the tab bar itself can be had by accessi
 * ```classForCell``` - Custom class to use for the tabs
 * ```classForDivider``` - Custom decorator view to use for tab separators
 
+## Changes
```
Comment (mattyohe) [inline README.md]: Thanks for the changes. Maybe we just need a CHANGELOG.md at the root?

File: Sources/ScrollingTabController.swift [hunk 2]
```diff
@@ -268,25 +195,18 @@ public class ScrollingTabController: UIViewController, UIScrollViewDelegate, UIC
...
-            let height = NSLayoutConstraint(item: child.view, attribute: .Height, relatedBy: .Equal, toItem: container, attribute: .Height, multiplier: 1.0, constant: 0.0)
-            let top = NSLayoutConstraint(item: child.view, attribute: .Top, relatedBy: .Equal, toItem: container, attribute: .Top, multiplier: 1.0, constant: 0.0)
-            let left = NSLayoutConstraint(item: child.view, attribute: .Left, relatedBy: .Equal, toItem: container, attribute: .Left, multiplier: 1.0, constant: 0.0)
-            
-            self.addChildViewController(child)
-            container.addSubview(child.view)
-            NSLayoutConstraint.activateConstraints([width, height, top, left])
-            child.didMoveToParentViewController(self)
-        }
+        self.addChildViewController(child)
```
Comment (mattyohe) [inline Sources/ScrollingTabController.swift]: ~~self~~

File: Sources/ScrollingTabController.swift [hunk 3]
```diff
@@ -349,15 +262,11 @@ public class ScrollingTabController: UIViewController, UIScrollViewDelegate, UIC
...
 
-        if let width = self.dataSource?.tabView(self, widthForCellAtIndex: indexPath.item) {
-            return CGSizeMake(width, height)
-        } else {
-            if let flowLayout = collectionViewLayout as? UICollectionViewFlowLayout {
-                return flowLayout.itemSize
-            }
-            
-            return CGSizeMake(100.0, height)
+        if let flowLayout = collectionViewLayout as? UICollectionViewFlowLayout {
```
Comment (mattyohe) [inline Sources/ScrollingTabController.swift]: Only concern here is that now dynamic sized tabs are not possible.

Comment (eriklamanna) [inline Sources/ScrollingTabController.swift]: Maybe leave a delegate for this case? Or should we just handle sizing dynamically?

**Pull request merged**